package.json

```diff
@@ -33,7 +32,7 @@
     "babel-plugin-system-import-transformer": "^2.4.0",
```

Comment:
should prob remove this and the devDep above it. we're no longer using system import or dynamic import anywhere, right?